yt_dlp/extractor/redbulltv.py

```diff
@@ -123,19 +119,11 @@ class RedBullEmbedIE(RedBullTVIE):  # XXX: Do not subclass from concrete IE
 
     def _real_extract(self, url):
         rrn_id = self._match_id(url)
-        asset_id = self._download_json(
-            'https://edge-graphql.crepo-production.redbullaws.com/v1/graphql',
-            rrn_id, headers={
-                'Accept': 'application/json',
-                'API-KEY': 'e90a1ff11335423998b100c929ecc866',
-            }, query={
-                'query': '''{
-  resource(id: "%s", enforceGeoBlocking: false) {
-    %s
-    %s
-  }
-}''' % (rrn_id, self._VIDEO_ESSENSE_TMPL % 'LiveVideo', self._VIDEO_ESSENSE_TMPL % 'VideoResource'),  # noqa: UP031
-            })['data']['resource']['videoEssence']['attributes']['assetId']
+        data = self._download_json(
+            url_or_request=f'https://api-player.redbull.com/rbcom/videoresource?videoId={rrn_id}',
+            video_id=rrn_id,
+        )
+        asset_id = data['assetId']
```

Comment:
1. For testing purpose, you can have some temp variables, but don't do it in the final code.
2. The location of parentheses is wrong. Read: [Long lines policy § CONTRIBUTING.md](https://github.com/yt-dlp/yt-dlp/blob/6ca23ffaa4663cb552f937f0b1e9769b66db11bd/CONTRIBUTING.md#long-lines-policy) and [Trailing parentheses § CONTRIBUTING.md](https://github.com/yt-dlp/yt-dlp/blob/6ca23ffaa4663cb552f937f0b1e9769b66db11bd/CONTRIBUTING.md#trailing-parentheses)